checklists/network_appdelivery_checklist.en.json

```diff
@@ -220,7 +220,7 @@
             "id": "A01.40",
             "ammp": true,
             "severity": "High",
-            "graph": "resources | where type == 'microsoft.network/frontdoorwebapplicationfirewallpolicies' | project policyName=name, policyId=id,policySku=sku.name, links=properties.securityPolicyLinks, enabledState=properties.policySettings.enabledState, mode=properties.policySettings.mode | mvexpand links | extend securityPolicy=links.id | extend securityPolicyParts=split(securityPolicy, '/') | extend profileId=strcat_array(array_slice(securityPolicyParts, 0, -3), '/') | project id=profileId, compliant=((enabledState=='Enabled') and (mode=='Prevention')), enabledState, mode",
+            "graph": "resources | where type == "microsoft.network/applicationgatewaywebapplicationfirewallpolicies" | extend compliant = (properties['policySettings']['mode'] == "Prevention")| where properties['policySettings']['mode'] == "Prevention" | distinct id, name, compliant",
```

Comment:
* Please use the case-insensitive operator `=~` instead of `==`.
* Please use single quotes for strings inside of the `graph` field, otherwise the JSON is invalid.
Thanks!